Upstream: No, this is on our telepathy being broken somehow.
Reason: This is a hotfix to not break everyone's GNOME Session over this.
Issue opened, see: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/2733

diff --git a/js/ui/components/telepathyClient.js b/js/ui/components/telepathyClient.js
index b4af438..109d324 100644
--- a/js/ui/components/telepathyClient.js
+++ b/js/ui/components/telepathyClient.js
@@ -18,7 +18,7 @@ const MessageTray = imports.ui.messageTray;
 const Params = imports.misc.params;
 const Util = imports.misc.util;
 
-const HAVE_TP = Tp != null && Tpl != null;
+const HAVE_TP = false;
 
 // See Notification.appendMessage
 var SCROLLBACK_IMMEDIATE_TIME = 3 * 60; // 3 minutes
